WebHostLib/options.py

```diff
@@ -84,11 +84,13 @@ def get_html_doc(option_type: type(Options.Option)) -> str:
                     "max": option.range_end,
                 }
 
-                if issubclass(option, Options.SpecialRange):
-                    game_options[option_name]["type"] = 'special_range'
+                if issubclass(option, Options.NamedRange):
```

Comment:
you changed stuff here without adjusting any of the js so i suspect this doesn't even work